.eslintrc

```diff
@@ -0,0 +1,60 @@
+{
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "prettier",
+    "universe"
+  ],
+  "parser": "@babel/eslint-parser",
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "parserOptions": {
+    "requireConfigFile": false,
+    "babelOptions": {
+      "presets": ["@babel/preset-react"]
+    },
+    "ecmaFeatures": {
+        "jsx": true
+    },
+    "ecmaVersion": 12,
```

Comment:
I'm curious, why default to 12..? what do we think about latest...?
There are so many options.. all the way to 15..

Reply:
Not sure, I grabbed this from another config. I'll set it to 15.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/719/commits/f41bdf37ba018948b4fe335f58d8270451592b42